### ISSUE TITLE
inline comments are not converted properly

### ISSUE THREAD

**Issue by gera2ld** [postcss/postcss-scss#79]
It seems that only inline comments outside rules are converted.

For example,
```scss
// comment1
a,
// comment2
b {
  color: red;
}
```
Expect
```css
/* comment1*/
a,
/* comment2*/
b {
  color: red;
}
```
but get
```css
/* comment1*/
a,
// comment2
b {
  color: red;
}
```

**Comment by ai** [comment]
Yeap. It is bug. I will try to fix it tomorrow

**Comment by ai** [comment]
Done 4ce59a2

**Comment by ai** [comment]
Released in 1.0.4. Thanks for the report. Happy Chinese New Year holiday month :D.

**Comment by gera2ld** [comment]
Thank you very much. 👍
